plugins/browser_auth.c

```diff
@@ -817,19 +814,18 @@ MDBUG_C_RETURN(Dbc, Dbc->Error.ReturnValue, &Dbc->Error);
   GError *error = NULL;
   secret_password_store_sync(JWT_CACHE_SCHEMA, SECRET_COLLECTION_DEFAULT,
                             SECURE_JWT_STORAGE_KEY, bac->token, NULL, &error,
-                            "email", bac->email,
                             NULL);
   if (error != NULL)
   {
     MDBUG_C_PRINT(Dbc, "%s FAILED", "secret_password_store_sync");
-    MADB_SetError(&Dbc->Error, error->code, error->message, 0);
+    MADB_SetError(&Dbc->Error, MADB_ERR_HY000, error->message, 0);
```

Comment:
I don't think that error returned by secret_password_store_sync matches the ODBC error codes

Reply:
you're right, that's why the previous version produced a wrong result (see https://memsql.atlassian.net/browse/PLAT-6231)

Reply:
Ou, I see.
I don't get used to the GitHub UI and thought that it was changed in the other way :)